src/middlewared/middlewared/utils/disks_/disk_class.py

```diff
@@ -134,6 +179,61 @@ def identifier(self) -> str:
         else:
             return f"{{devicename}}{self.name}"
 
+    @property
+    def temp(self) -> TempEntry:
```

Comment:
If it is a command, should we have it as a class method so it’s clear that reading it takes some time and has side effects?

Reply:
I'm not sure why I'd use a `@classmethod`? I could make it a standard method instead of a `@property`.

Reply:
Yes, standard method is what I meant

Reply:
Okay, fixed.